### DISCUSSION TITLE
Proform 5000 incline using Zwift

### DISCUSSION THREAD

**Discussion by Shaunbuk66** [cagnulein/qdomyos-zwift#1570] [General]
Hello
I purchased the app on the play store but cannot get it to connect to my treadmill which is a Proform 5000 model reading the zwift forums  coming on here may help.

I would prefer to do all on my laptop if possible so i have just my treadmill and laptop nothing else but if need to i will use phone as well

Thanks in advance for any assistance

**Comment by cagnulein** [comment]
Hi! Try this, in the qz settings, treadmill, proform section, enable one
switch at a time leaving only one enabled and restart qz after each test
Let me know if you find a setup that works, otherwise I have to add the
compatibility with yours
Let me know

Il giorno dom 16 lug 2023 alle 19:46 Shaunbuk66 ***@***.***>
ha scritto:

> Hello
> I purchased the app on the play store but cannot get it to connect to my
> treadmill which is a Proform 5000 model reading the zwift forums coming on
> here may help.
>
> I would prefer to do all on my laptop if possible so i have just my
> treadmill and laptop nothing else but if need to i will use phone as well
>
> Thanks in advance for any assistance
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/1570>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWAFIXBZ7RFMBRDLNDDXQQSHRANCNFSM6AAAAAA2MCKO64>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Reply by Shaunbuk66** [reply]
will do it now and return with findings

**Reply by Shaunbuk66** [reply]
Hi I have done as requested and it has still not found the Proform 5000 Treadmill it found my wahoo watch though :-)

**Reply by cagnulein** [reply]
What do you mean? You don't see the tiles at all in qz? If so send me a nrfconnect screenshot about it (nrfconnect is a free app on the store)
Thanks

**Reply by Shaunbuk66** [reply]
no tiles at all when i press padlock it says you can move tiles but there are no tiles

**Reply by cagnulein** [reply]
So send me a screenshot of and qz and nrfconnect as I said , probably it has a different Bluetooth name
Let me know

**Reply by Shaunbuk66** [reply]
sorry i meant the tiles are there when i turn on my wahoo kickr bike no problem but they vanish when i turn it and i cant find the Proform 5000

**Reply by cagnulein** [reply]
Ok because qz is also compatible with your bike. Please send a nrfconnect screenshot about your treadmill

**Reply by Shaunbuk66** [reply]
sorry i dont understand how to do a screenshot with the nrfconnect there is nothing to send the QZ fitness does not find the treadmill so there is nothing to send

**Reply by cagnulein** [reply]
![image](https://github.com/cagnulein/qdomyos-zwift/assets/48216/af9167c5-8518-42ca-ba51-8d3789d7cbba)

Send me something like this showing your treadmill on here (treadmill turned on)

**Reply by Shaunbuk66** [reply]
ok

**Reply by Shaunbuk66** [reply]
![Screenshot_20230716_194505_nRF Connect](https://github.com/cagnulein/qdomyos-zwift/assets/139704988/ddc2d6f6-4ffb-4098-88e6-e6f389cf15c6)

**Reply by cagnulein** [reply]
I don't see any treadmill here. Are you sure your treadmill is power on, near to you and with Bluetooth enabled? Are you sure your treadmill is Bluetooth capable? (For example does it work with the proform app?)
Let me know

**Comment by Shaunbuk66** [comment]
i am checking but it has bluetooth audio which i attach my phone to and my head phones 
but you are correct it is not there so i will turn it off and on again

**Reply by cagnulein** [reply]
So maybe it has Bluetooth for only audio (Bluetooth 2.0). Do you ever use an app with it?

**Reply by cagnulein** [reply]
@Shaunbuk66 any news?
